### PR TITLE
Fix code typo

### PR DESCRIPTION
**Pull request by SuperOP535**

### CODE REVIEW

File: examples/chatterbox.js [hunk 1]
```diff
@@ -158,9 +158,8 @@ bot.on('playerJoined', (player) => {
   }
 })
 bot.on('playerLeft', (player) => {
-  if (player.username === bot.username) {
+  if (player.username === bot.username) return;
     bot.chat(`Bye ${player.username}`)
```
Comment (roblabla) [inline examples/chatterbox.js]: Could you remove two spaces here, so it's properly indented ?

Comment (SuperOP535) [inline examples/chatterbox.js]: Uhm could you tell me where exact?

Comment (roblabla) [inline examples/chatterbox.js]: bot.chat, remove two spaces before it :)

**Comment by SuperOP535** [comment]
I made a bonus and fixed playerJoined double space too ;)

**Comment by roblabla** [comment]
No, the double space in playerJoined is OK.

When something is between braces, it should be intended. Basically, 

```
if (something) {
  ThatHappensIfSomething // So we make it visually "under" something
}
```

But

```
if (something) return;
ElseThisHappens // This doesn't happen if something, so we don't put it "under" it. We don't put spaces
```

**Comment by SuperOP535** [comment]
Well, I have been coding JavaScript in a long timeand I did not notice that ._.

**Comment by SuperOP535** [comment]
Cool now I have the Contributor tag

**Pull request merged**